pkg/apis/minio.min.io/v2/types.go

```diff
@@ -514,6 +528,11 @@ type Pool struct {
 	//
 	// +optional
 	SecurityContext *corev1.PodSecurityContext `json:"securityContext,omitempty"`
+	// *Optional* +
+	//
+	// Specify custom labels and annotations to append to the Pool.
+	// +optional
+	PoolMetadata *PoolMetadata `json:"poolMetadata,omitempty"`
```

Comment:
In all the other places where we allow `labels` and `annotations` where simply put them at struct level (PrometheusConfig, LogConfig, LogDbConfig, KESConfig) so I think it's better to place the labels and annotations fields here and just call them `labels` and `annotations` for consistency

Reply:
@dvaldivia thanks! 

Changed here as suggested and already rebased for 1 commit only.